globus_automate_client/cli/helpers.py

```diff
@@ -9,7 +10,11 @@
 )
 
 
-def format_and_echo(result: Union[GlobusHTTPResponse, str], verbose=False) -> None:
+def format_and_echo(
+    result: Union[GlobusHTTPResponse, str],
+    dumper: Callable = lambda result, **args: json.dumps(result, indent=4, sort_keys=True),
```

Comment:
After staring at it for a second, I don't see an actual issue with this, but my python senses would be tingling less if this was an `Optional[Callable]` defaulting to None and updated to this lambda in the body if None. Or if there was a small `dump_to_json` function
```python
def dump_to_json(result, *args, **kwargs):
    return json.dumps(result, indent=4, sort_keys=True)
```
and this defaults to `dumper: Callable = dump_to_json`